recipes/ligo-gracedb/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "ligo-gracedb" %}
+{% set version = "1.28" %}
+{% set sha256 = "a14867eef76f890b339f500fe3d1fb1dc94f3d1b90ad8e148a5ce4f1b5f9243a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
```

Comment:
Could you please just put a comment here explaining the reasons why you patched it ?

Reply:
@marcelotrevisani see d48895b

Reply:
Thanks! :)
Looks good to me, after CI I will merge that